components/permissions_gates/any_team_permission_gate/any_team_permission_gate.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+export interface Props{
+
+    /**
+     * Permissions enough to pass the gate (binary OR)
+     */
+    permissions: string[],
+
+    /**
+     * Has permission
+     */
+    hasPermission?: boolean,
+
+    /**
+     * Invert the permission (used for else)
+     */
+    invert?: boolean,
```

Comment:
Same with this one.

Reply:
This one is optional because it has a default in the original component. The default makes no sense if the prop is required.